### PR TITLE
chore: add owners

### PR DESCRIPTION
**Pull request by whalecold**
**What this PR does / why we need it**:

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**:

### CODE REVIEW

File: OWNERS [hunk 1]
```diff
@@ -4,7 +4,15 @@ approvers:
   - ingress-nginx-admins
   - ingress-nginx-maintainers
   - ElvinEfendi
+  - whalecold
```
Comment (supereagle) [inline OWNERS:7]: Only need to add core maintainers into approvers.

Comment (whalecold) [reply OWNERS:7]: Done.

**Review by supereagle — APPROVED** [review]
/lgtm

**Comment by supereagle** [comment]
Manually merge the PR for adding owners.

**Pull request merged**